Cargo.toml

```diff
@@ -11,3 +11,6 @@ docker-api = "0.9.1"
 tokio = { version = "1.0", features = ["full"] }
 futures-util = "0.3"
 tar = "0.4"
+predicates = "2.1.1"
+assert_cmd = "2.0.4"
+rand = "0.8.5"
```

Comment:
We might be able to add these under `dev-dependencies` since they are only for testing.

Reply:
Yep! Went ahead and fix this.